### PR TITLE
Fixes #4

### LINKED ISSUE #4: Remove toasts
We should remove the toasts, at least in the release version

### PR DESCRIPTION
**Pull request by SrMonedero**
Removes the annoying toasts from the auth feedback

### CODE REVIEW

File: app/src/main/java/firebasebarcelona/firepadel/ui/courts/view/CourtListFragment.java [hunk 1]
```diff
@@ -141,7 +141,7 @@ public void onAuthStateChanged(@NonNull FirebaseAuth auth) {
         if (user != null) {
           saveLocalPlayer(user);
         } else {
-          Toast.makeText(getActivity(), "Auth fail", Toast.LENGTH_SHORT).show();
+          //TODO show some kind of error
```
Comment (jcondominas) [inline app/src/main/java/firebasebarcelona/firepadel/ui/courts/view/CourtListFragment.java]: TODO don't work. We do not have methodoloy for TODO. 

I rather delete them, and write down the task to design error msg feedback than leave a TODO here.

**Review by jcondominas — CHANGES_REQUESTED** [review]
Could you remove TODO, what do you think?

File: app/src/main/java/firebasebarcelona/firepadel/ui/courts/view/CourtListFragment.java (hunk 1 above)
Comment (SrMonedero) [reply app/src/main/java/firebasebarcelona/firepadel/ui/courts/view/CourtListFragment.java]: Well, it's a todo, so we have to do it. We don't need any methodology, and I don't have the time to do this design right now. So I prefer to delete a bad feedback from de app and put a TODO

Comment (jcondominas) [reply app/src/main/java/firebasebarcelona/firepadel/ui/courts/view/CourtListFragment.java]: Yes, I mean, we could use issue to keep track of what we have to do, or projects, or trello, or whatever. But keep the code clean and free of comments and TODO

Comment (SrMonedero) [reply app/src/main/java/firebasebarcelona/firepadel/ui/courts/view/CourtListFragment.java]: ok, agree

Comment (SrMonedero) [reply app/src/main/java/firebasebarcelona/firepadel/ui/courts/view/CourtListFragment.java]: @jcondominas changed

**Review by jcondominas — APPROVED** [review]

**Pull request merged**